src/models/data.py

```diff
@@ -420,6 +443,33 @@ def __init__(self, loader: DataLoader) -> None:
     def __iter__(self):
         return self
 
+    def _get_prev_y_var(
```

Comment:
is there any way we can pass in the mean of all the previous months? So in the train data this will be either 1 (for 1982) or 29 for 2010 (29 previous years of that month).

In the test it would just be the mean of the training data (like a climatology)

Reply:
maybe for a future PR?

Reply:
maybe that should be `get_mean_y_value` or something

Reply:
yea. We sort of have that with the yearly_aggs, but it could definitely be revisited.